cmd/chartsvc/handler.go

```diff
@@ -200,6 +204,90 @@ func getChartVersionValues(w http.ResponseWriter, req *http.Request, params Para
 	w.Write([]byte(files.Values))
 }
 
+func isChartContained(chartData *models.Chart, chartMetadata *chart.Metadata) bool {
```

Comment:
Is chart contained or maybe a better name would be isSameChart? That's what you are checking no?